Sources/Apodini/Semantic Model Builder/Protobuffer/ProtobufferInterfaceExporter.swift

```diff
@@ -10,7 +10,9 @@ class ProtobufferInterfaceExporter: StaticInterfaceExporter {
         let message: String
     }
 
-    internal enum Builder {}
+    enum Builder {
+        static var variableWidthIntegerPreference: Int?
```

Comment:
Could we reuse `Option` (or `IntWidthOption` 😁) instead of the `Int` type? To enforce that only valid values (32 and 64) can be set?